contracts/sg721-updatable/src/state.rs

```diff
@@ -16,8 +16,8 @@ pub struct CollectionInfo<T> {
 pub struct RoyaltyInfo {
     pub payment_address: Addr,
     pub share: Decimal,
-    pub updated_at: Option<Timestamp>,
```

Comment:
just want to check with @MightOfOaks to make sure this won't break on read

Reply:
Earlier the code was in sync with a v2 update that we abandoned and was never deployed on chain. So, the v1 contracts that are currently on chain do not have an `updated_at` attribute in their CollectionInfo.RoyaltyInfo.

The PR basically reverts all the previous changes regarding royalty updates and keeps the update_time separately similar to how it's done for the v2 contracts.